Automattic/jetpack
# Stats: make nudges collapsible

#### Changes proposed in this Pull Request:
* Add the ability to collapse the nudges UI.

#### Jetpack product discussion
pbNhbs-189-p2#comment-3564

#### Does this pull request change what data or activity we track or use?
* no

#### Testing instructions:
1. Install this branch of Jetpack. Activate Jetpack and connect (a site or user connection is fine).
2. In wp-admin, navigate to Jetpack -> Site Stats.
    * If the stats chart isn't available yet, the collapse toggle isn't displayed.
    * If the stats chart is available, the collapse toggle is displayed.
3. The Nudges UI should be expanded by default.
4. Collapse the nudges UI by clicking on the triangle icon.
5. Reload the page to verify that the collapsed state persists.
6. Expand the nudges UI by clicking on the triangle icon.
7. Reload the page to verify that the expanded state persists.

##### Update Jetpack
If Jetpack is updated, the nudges collapse state will be reset.
1. In wp-admin, navigate to Jetpack -> Site Stats. Collapse the nudges.
2. Increase the Jetpack version in the `jetpack.php` file [here](https://github.com/Automattic/jetpack/blob/master/projects/plugins/jetpack/jetpack.php#L35).
3. In wp-admin, navigate to Jetpack -> Site Stats. Verify that the nudges are expanded.

#### Screenshots

##### Expanded

<img width="1015" alt="Screen%20Shot%20on%202021-08-23%20at%2015-23-23" src="https://user-images.githubusercontent.com/50059399/130507234-7d685f46-2de5-4baa-addd-2d86df533df8.png">

##### Collapsed

<img width="1016" alt="Screen%20Shot%20on%202021-08-23%20at%2015-23-49" src="https://user-images.githubusercontent.com/50059399/130507288-a147734d-5543-495b-9aae-6a1f1a8077eb.png">
